.zenodo.json

```diff
@@ -136,11 +139,17 @@
       "name": "Thirion, Bertrand",
       "orcid": "0000-0001-5018-7895"
     },
+    {
+      "name": "Chadwick Boulay"
+    },
     {
       "affiliation": "Rutgers University",
       "name": "Kleinschmidt, Dave F.",
       "orcid": "0000-0002-7442-2762"
     },
+    {
+      "name": "Erin W Dickie"
```

Comment:
@edickie  Do you want to be an author on the Zenodo, and if so, what affiliation, name and [ORCID](http://orcid.org/) should we use?